### PR TITLE
Initial refactoring for standardizing linting and testing

### PR DESCRIPTION
**Pull request by cmin764**
Dependency of https://github.com/robocorp/java-access-bridge-wrapper/pull/14

Adds a thin layer of not too opinionated formatting (from our configuration intervention/deviation PoV):
- Linting: _isort_, _black_, _flake8_/_pep8_ (exception for the line length 88 -> 120 chars; TBD)
- Beta **1.0.0** release with package updates included
- Applies current rules into formatting through `inv lint -a` command

❗ By any means this doesn't and shouldn't change behaviour, it's only a cosmetic refactoring aided by Invoke through Poetry commands.

### ToDo
- [x] Refactor the entire test with _pytest_. (and fixtures for the JAB wrapper and context tree)
- [x] Add `build` and `publish` (CI/prod) Invoke commands.
- [x] Add simple README section with:
   - [x] How to quickly isolate the env preparation with _pyenv_. (so Poetry and Invoke are made available)
   - [x] How to setup Poetry config and credentials. (for CI/prod package publishing)

Check updated README [Development](https://github.com/robocorp/java-access-bridge-wrapper/tree/refactor/linting-testing#development) instructions

### CODE REVIEW

File: config/flake8 [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[flake8]
+ignore = W503, E203, E501
```
Comment (cmin764) [inline config/flake8:2]: Replace with ignore names. (meaning)

File: config/flake8
Comment (cmin764) [inline config/flake8:1]: Add CI pipeline. (for linting)

Comment (cmin764) [reply config/flake8:1]: Maybe @mikahanninen or @mmokko can give it a hand here in my absence. We just need to copy the essential from this pipeline [example](https://github.com/robocorp/rpaframework/blob/master/.github/workflows/main.yaml) where we're interested only into running linting (`inv lint`) on a Windows env with Py3.10 for now.

Comment (mmokko) [reply config/flake8:1]: Sure I can add the pipeline.

**Review by mmokko — APPROVED** [review]
Looks good, I'll add the pipeline and merge.

**Pull request merged**